### ISSUE TITLE
Un'Goro Issues

### ISSUE THREAD

**Issue by DominusMaximus** [webadict/metastone#15]
I'll start posting any issues I find here. 

- Quests aren't automatically appearing as the leftmost card in a player's mulligan. I noticed this wasn't on your checklist. Will this be implemented?

**Comment by webadict** [comment]
This will eventually be implemented. I haven't added this feature just yet, but it'll be an Attribute to put on cards.

**Comment by DominusMaximus** [comment]
**1.)** If I play Living Mana with 10/10 mana available, my mana will end up as -2/3. Cards such as Felguard, Darnassus Aspirant, and Blastcrystal Potion destroy empty mana crystals first. I'm assuming this will also apply to Living Mana. If this is the case, the resulting mana count should be 3/3. 

**2.)** Temporary mana gained through cards such as Innervate or The Coin should never be targeted by mana destroying effects. For example, if I have 0/0 mana and a Darnassus Aspirant on board, and then play Innervate and somehow destroy my Aspirant without using the Innervated mana, my mana count should remain at 2/0.

**Comment by webadict** [comment]
Yeah, it's weird because Living Mana was actually working at one point... And then it stopped working. I'll try to find what I broke.

**Comment by DominusMaximus** [comment]
Interesting Un'Goro interactions: https://www.youtube.com/watch?v=2hA044bnsOk

**Comment by DominusMaximus** [comment]
1.) The 5 and 10 mana versions of Icecap are randomly targeting a minion more than once while additional targets are available.

2.) Redemption should summon its minion at the board position that was occupied by the minion that triggered it.

3.) Molten Blade isn't retaining weapon buffs from cards such as Hobart Grapplehammer and Grimestreet Pawnbroker. This is negatively affecting my ability to go face.

4.) In Sandbox mode, if you modify your hand after a Molten Blade transforms, it will no longer transform at the beginning of your turn. I don't know if this would affect anything in Play Mode.

**Comment by webadict** [comment]
1. Probably needs to have a different spell used for it. I think I've got ideas.
2. ... Huh, should be doable. Probably low on priorities, though.
3. Issue with the Buff system at the moment. Needs to be fixed. The same happens to Shifter Zerus.
4. Not an issue with that, it's an issue with how the Cards are saved. It shouldn't have any effect in Play Mode.

**Comment by ikhaliq15** [comment]
Also, btw, the priest quest is Awaken the Makers, not Awaken the Masters.

**Comment by webadict** [comment]
Good catch. Thanks.

**Comment by DominusMaximus** [comment]
1.) Crackling Razormaw is Adapting itself.

2.) Primordial Glyph has an unnecessary forward slash on line 9.

**Comment by webadict** [comment]
Anything with a forward slash is on purpose, as that card does not work correctly.

**Comment by DominusMaximus** [comment]
- Sunkeeper Tarim crashes the game when played. Seems to be due to the formatting of the MetaSpell portion.

**Comment by DominusMaximus** [comment]
- Tar Creeper isn't updating its attack once the turn ends. It will update its attack only after certain actions (playing a card, for example). Attacking it or with it will not update its attack.

**Comment by DominusMaximus** [comment]
- Unbound Elemental is being buffed regardless of whether a played card has Overload. Using Crowd Favorite's template seems to work.
